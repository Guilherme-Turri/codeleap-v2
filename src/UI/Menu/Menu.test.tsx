import { screen } from '@testing-library/react'
import { Menu } from '.'
import { componentRender } from '../../render/componentRender'

describe('</Menu>', () => {
  it('should render on screen', () => {
    const { container } = componentRender(<Menu />)
    expect(screen.getByTestId('MenuOutline')).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  })
})