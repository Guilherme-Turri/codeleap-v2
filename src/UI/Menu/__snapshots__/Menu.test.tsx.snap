// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`</Menu> should render on screen 1`] = `
.c1 {
  display: inline-block;
  vertical-align: middle;
  overflow: hidden;
}

.c0 {
  color: #00CBA9;
  width: 100%;
  height: 100%;
}

<div>
  <div
    class="c0"
    data-testid="MenuOutline"
  >
    <svg
      aria-hidden="true"
      class="c1"
      fill="currentColor"
      focusable="false"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <g
        data-name="Layer 2"
      >
        <g
          data-name="menu"
        >
          <rect
            height="2"
            rx="0.95"
            ry="0.95"
            width="18"
            x="3"
            y="11"
          />
          <rect
            height="2"
            rx="0.95"
            ry="0.95"
            width="18"
            x="3"
            y="16"
          />
          <rect
            height="2"
            rx="0.95"
            ry="0.95"
            width="18"
            x="3"
            y="6"
          />
        </g>
      </g>
    </svg>
  </div>
</div>
`;
